.github/workflows/devpackpublish.yml

```diff
@@ -3,7 +3,13 @@ on:
   push:
     branches:
       - develop
-      - tw-851-git-push
+    paths:
+      - '**/*.ts'
+      - 'package.json'
+      - '!**/*.spec.ts'
+      - '!**/__integrationtests__'
+      - '!**/__mocks__'
+      - '!docs/**'
```

Comment:
will only build if we change `.ts` files or the `package.json`, but not if the files are `.spec.ts` or in `__integrationtests__`, `__mocks__` or `docs`

Reply:
shall I add anything else?

Reply:
No, I think it is all.